os/tools/configure.sh

```diff
@@ -245,4 +245,45 @@ if [ "X${defappdir}" = "Xy" ]; then
     echo "CONFIG_APPS_DIR=\"$posappdir\"" >> "${dest_config}"
   fi
 fi
+
+# Check mandatory package to configuration set
+IOTIVITY=`grep CONFIG_ENABLE_IOTIVITY=y ${dest_config}`
+if [ ! -z ${IOTIVITY} ]; then
```

Comment:
Just curious, Is ! -z {var} better that -n {var}  or same?

Reply:
No. same